src/static/helpers/slasHelper.ts

```diff
@@ -298,6 +373,25 @@ export function refreshAccessToken(
     channel_id: slasClient.clientConfig.parameters.siteId,
   };
 
+  if (credentials && credentials.clientSecret) {
+    /* istanbul ignore next */
+    if (isBrowser) {
+      // we intentionally have warning here
+      // eslint-disable-next-line
+      console.warn(warningSlasSecretMsg);
+    }
```

Comment:
Is there a way to suppress this warning if we are using a placeholder value for the client secret or will it always appear as long as this function runs client side.

Reply:
it will always appear as long as the func runs on client side. I don't use any place holder value here in this library since I find it unnecessary.